tegra/tegra.c

```diff
@@ -206,7 +206,7 @@ int drm_tegra_bo_unref(struct drm_tegra_bo *bo)
 		return -EINVAL;
 
 	if (!atomic_dec_and_test(&bo->ref))
-		return 0;
+		return atomic_read(&bo->ref);
```

Comment:
Well, only if compiled with a debug-version of libdrm, which I'm guessing people do exactly *because* they want things like this...